controllers/drcluster_controller.go

```diff
@@ -1330,7 +1330,7 @@ func (u *drclusterInstance) createNFManifestWork(targetCluster *ramen.DRCluster,
 	annotations[DRClusterNameAnnotation] = u.object.Name
```

Comment:
What is NF manifest?

Reply:
NF manifest work is used for managing networkfence resource